tests/acceptance/test_acceptance/__init__.py

```diff
@@ -0,0 +1 @@
+# __init__ is empty
```

Comment:
Nit, this module name feels redundant. Should we name it `tests/acceptance/test`? Also is this string needed?